server/gabble/chat/primitive_events.go

```diff
@@ -4,26 +4,25 @@ import (
 	"log"
 
 	socketio "github.com/googollee/go-socket.io"
-	config "github.com/whizzes/gabble/server/src/chat/config"
```

Comment:
Moved the `config` module to `gabble` package in order to have a global config for all the packages

Reply:
Intended to avoid using multiple configurations defined in different packages which can lead to a cyclic reference